modules/administration-guide/partials/proc_configuring-bitbucket-server-oauth1.adoc

```diff
@@ -41,7 +41,7 @@ openssl rand -base64 24 > __<bitbucket_server_consumer_key>__
 openssl rand -base64 24 > __<bitbucket_shared_secret>__
 ----
 
-. Create a Kubernetes Secret in {prod-short} namespace containing the consumer and private keys.
+. Create a {orch-name} Secret in {prod-short} namespace containing the consumer and private keys.
```

Comment:
Will resolve as `a OpenShift`... Can we get away with `your {orch-name} secret`?

Reply:
one?